sympy/printing/pretty/pretty.py

```diff
@@ -1522,6 +1521,30 @@ def _print_Contains(self, e):
         else:
             return prettyForm(sstr(e))
 
+    def _print_SeqFormula(self, s):
```

Comment:
The solution to this is not to use `isympy` for debugging the code then. The normal string printer works just fine.